python/ray/signature.py

```diff
@@ -27,6 +29,39 @@
 """
 
 
+def get_signature_params(func):
+    """Get signature parameters
+    Probably the hackiest thing imaginable to support Cython built-ins. Create
```

Comment:
Hm, if there is a better way to do this, consider doing it that way (I don't see one right now); otherwise maybe make the comment sound less terrible, given all things I've seen people do with (to) Python, this is actually not *that* hacky ;)

Reply:
Hah! I'll admit the comment was more CYA, so if you guys are OK with it, then I'll reword the comment since I don't have a better answer. There's been some chatter in the Cython community ([one example](https://groups.google.com/forum/#!topic/cython-users/0k1GDVYVmUg)), but it seems that they're waiting on better Python support before doing anything about it.